src/components/Books.js

```diff
@@ -0,0 +1,22 @@
+import React, { useState } from 'react';
+import Book from './Book';
+import NewBook from './AddBook';
+
+const Books = () => {
+  const [book] = useState({
+    title: 'Book 1',
+    author: 'Author 1',
+  });
```

Comment:
[OPTIONAL]
- [ ] I would suggest you use an array of books as a placeholder instead of a single book